analyzer/src/main/kotlin/curation/OrtConfigPackageCurationProvider.kt

```diff
@@ -61,10 +61,8 @@ open class OrtConfigPackageCurationProvider : PackageCurationProvider {
         }
     }
 
-    override fun getCurationsFor(packages: Collection<Package>): Map<Identifier, List<PackageCuration>> =
-        packages.mapNotNull { pkg ->
-            getCurationsFor(pkg.id).takeUnless { it.isEmpty() }?.let { pkg.id to it }
-        }.toMap()
+    override fun getCurationsFor(packages: Collection<Package>): List<PackageCuration> =
+        packages.flatMap { pkg -> getCurationsFor(pkg.id) }.distinct()
```

Comment:
I believe `distinct()` should not be added here to more closely resemble the previous code, and because it would be dropped in the next commit anyway.